lib/simplifyExpression/arithmeticSearch/index.js

```diff
@@ -22,7 +22,7 @@ function arithmetic(node) {
   // we want to eval each arg so unary minuses around constant nodes become
   // constant nodes with negative values
   node.args.forEach((arg, i) => {
-    node.args[i] = Node.Creator.constant(arg.eval());
+    node.args[i] = Node.Creator.constant(arg.eval().toString(), "number");
```

Comment:
Could you set it up so that the string "number" isn't created here? to avoid typos and stuff

Maybe we could have an enum somewhere... or could create multiple Node.Creator constructors for each case? idk

Also I see that you always set it to "number". Would we ever want it to be something else?